scholia/app/static/scholia.js

```diff
@@ -218,6 +217,14 @@ function sparqlToResponse(sparql, doneCallback) {
 }
 
 
+function sparqlToResponse(sparql, doneCallback) {
```

Comment:
I am trying to understand this. As far as I can determine you do not have polymorphic functions in Javascript, so the second definition of `sparqlToResponse` will overwrite the first one? Or what am I missing?

Reply:
```
$ node -e "function f(a, b) { console.log('two arguments'); } function f(a) { console.log('one argument'); } f(1, 2);"
one argument
```

Reply:
@fnielsen, good catch. Are you okay to name the new one `sparqlToResponse`? If not, what name should it be?

Reply:
Patch pushed: https://github.com/WDscholia/scholia/pull/2515/commits/4f593ae28d48f969e78f5849c44edf13bbafda6c

Reply:
@fnielsen, what do you think?